automation_tools/__init__.py

```diff
@@ -244,6 +244,9 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
     if len(hostname) == 0:
         print('Was not possible to fetch hostname information')
         sys.exit(1)
+    if '.' not in hostname:
```

Comment:
Better to have separated otherwise will be required to repeat `hostname.split('.', 1)[1]`